src/app/screens/HomeTabs/Shows/Shows.tsx

```diff
@@ -17,17 +17,23 @@ export const Shows = () => {
   return (
     <TabsFlatList
       data={shows}
-      renderItem={({ item: show }) => (
-        <Touchable
-          onPress={() => {
-            navigation.navigate("ShowTabs", {
-              slug: show.slug || "",
-            })
-          }}
-        >
-          <ShowListItem show={show} />
-        </Touchable>
-      )}
+      renderItem={({ item: show }) => {
+        if (show.artworksCount! > 0) {
```

Comment:
```suggestion
        if (show.artworksCount && show.artworksCount > 0) {
```
small suggestion to avoid the !

Reply:
Its basically the same thing, right? isn't it?